gofsutil_utils.go

```diff
@@ -0,0 +1,51 @@
+package gofsutil
+
+import (
+	"errors"
+	"path/filepath"
+	"regexp"
+)
+
+func validatePath(path string) error {
+	if path == "/" {
+		return errors.New("Path: " + path + " is invalid")
+	}
+
+	return nil
+}
+
+func validateFsType(fsType string) error {
+	if fsType != "ext4" && fsType != "ext3" &&
+		fsType != "xfs" && fsType != "nfs" {
+		return errors.New("FsType: " + fsType + " is invalid")
+	}
+
+	return nil
+}
+
+func validateMountOptions(mountOptions ...string) error {
+	for _, opt := range mountOptions {
+		matched, err := regexp.Match(`[\w]+[=]*[\w]*`, []byte(opt))
```

Comment:
Please add a comment describing the regex with a sample so that it is easy for people to understand the validation

Reply:
samples added